lib/order.rb

```diff
@@ -0,0 +1,65 @@
+class Order
+
+  attr_reader :current_order, :bill, :string_summary
+
+  def initialize
+    @menu = Menu.new
+    @current_order = []
+    @bill = 0
+    @string_summary = ''
```

Comment:
Is this required? Could the string_summary be derived from the `@current_order` when it was needed?